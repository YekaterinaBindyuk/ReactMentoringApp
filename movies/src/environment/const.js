export const BASE_IMG_CLASSNAME = 'img';
export const MOVIE_LIST_ITEM_COVER = 'small';
export const MOVIE_COVER = 'large';
export const MAIN_LOGO = 'http://localhost:8080/src/images/logo.png';
export const SEARCH_BUTTON = 'Search';
export const TITLE_TYPE_BUTTON = 'title';
export const GENRE_TYPE_BUTTON = 'genre';
export const WARNING = 'Something went wrong';
export const RATING_TYPE_BUTTON = 'rating';
export const RELEASE_TYPE_BUTTON = 'release date';
export const SORTING_TEXT = 'Sort by: ';
export const SEARCH_TEXT = 'Search by: ';
export const NO_FILMS_WARNING = 'No films found';
export const MOVIES_COUNT_TEXT = ' films found:';
export const RECOMMENDED_MOVIES = 'Recommended movies';
export const MAIN_PAGE = '/';
export const REST_URL = 'https://reactjs-cdp.herokuapp.com/';
export const MOVIES_URL = REST_URL + 'movies';
