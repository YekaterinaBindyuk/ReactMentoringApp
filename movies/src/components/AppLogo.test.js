import React from "react";
import renderer from "react-test-renderer";
import { MAIN_LOGO } from "../environment/const";
import { MAIN_PAGE } from "../environment/const";

it("renders correctly", () => {
  const tree = renderer
    .create(
     <a href={MAIN_PAGE}>
      <img src={MAIN_LOGO} alt="logo" className="app-logo"/>
    </a>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
