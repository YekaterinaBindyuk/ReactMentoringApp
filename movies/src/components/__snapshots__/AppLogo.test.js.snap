// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  href="/"
>
  <img
    alt="logo"
    className="app-logo"
    src="/static/images/logo.png"
  />
</a>
`;
